voting-hir/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use jormungandr_lib::{crypto::account::Identifier, interfaces::Value};
+
+pub type VotingGroup = String;
+
+/// Define High Level Intermediate Representation (HIR) for voting
+/// entities in the Catalyst ecosystem.
+///
+/// This is intended as a high level description of the setup, which is not
+/// enough on its own to spin a blockchain, but it's slimmer, easier to understand
+/// and free from implementation constraints.
+///
+/// You can roughly read this as
+/// "voting_key will participate in this voting round with role voting_group and will have voting_power influence"
+pub struct VotingHIR {
+    pub voting_key: Identifier,
+    /// Voting group this key belongs to.
+    /// If this key belong to multiple voting groups, multiple records for the same
```

Comment:
But why does not to store it inside an array if we have multiple records ?

Reply:
I don't get what you are suggesting

Reply:
I am talking about to replace `pub voting_group: VotingGroup,` to the `pub voting_group: Vec<VotingGroup>,`, so to not create a one record for the `voting_key`

Reply:
but those voting groups might have different voting power, so you should also have a `Vec` of voting power, but then the relation between the two is not so tight anyway. Also, in the beginning we're probably only going to have one group per voter anyway, and this seems like the simplest solution to ma that is both not an overkill for now and flexible enough in the future.